cmd/checkconfig/testdata/combined.yaml

```diff
@@ -23,6 +23,10 @@ deck:
           - (FAIL|Failure \[)\b
           - panic\b
           - ^E\d{4} \d\d:\d\d:\d\d\.\d\d\d]
+          iframe_sandbox_permissions:
+          - allow-scripts
+          - allow-popups
+          - allow-popups-to-escape-sandbox
```

Comment:
Updated this as it looked like a sane idea to do if this is supposed to demostrate a full configuration file example.